apps/full-site-editing/full-site-editing-plugin/mailerlite/subscriber-popup.php

```diff
@@ -0,0 +1,130 @@
+	/**
+	 * WPCOM_Widget_Mailerlite constructor.
+	 */
+	public function __construct() {
+		parent::__construct(
+			'wpcom-mailerlite',
+			/** This filter is documented in modules/widgets/facebook-likebox.php */
+			apply_filters( 'jetpack_widget_name', __( 'Mailerlite subscriber popup', 'jetpack' ) ),
+			array(
+				'classname'                   => 'widget_mailerlite',
+				'description'                 => __( 'Display Mailerlite subscriber popup', 'jetpack' ),
+				'customize_selective_refresh' => true,
+			)
+		);
+	}
+
+	/**
+	 * Output the widget.
+	 *
+	 * @param array $args Display arguments including 'before_title', 'after_title', 'before_widget', and 'after_widget'.
+	 * @param array $instance - The settings for the particular instance of the widget.
+	 */
+	public function widget( $args, $instance ) {
+		/** This action is documented in modules/widgets/gravatar-profile.php */
+		do_action( 'jetpack_stats_extra', 'widget_view', 'mailerlite' );
+
+		if ( empty( $instance['account'] ) || empty( $instance['uuid'] ) ) {
+			if ( current_user_can( 'edit_theme_options' ) ) {
+				echo $args['before_widget']; //phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+				echo '<p>' . sprintf(
+					wp_kses(
+						/* translators: %1$s - URL to manage the widget, %2$s - documentation URL. */
+						__( 'You need to enter your numeric account ID and UUID for the <a href="%1$s">Mailerlite Widget</a> to work correctly. <a href="%2$s" target="_blank">Full instructions</a>.', 'jetpack' ),
+						array(
+							'a' => array(
+								'href'  => array(),
+								'title' => array(),
+							),
+						)
+					),
+					esc_url( admin_url( 'widgets.php' ) ),
+					'https://wordpress.com/support/widgets/mailerlite'
+				) . '</p>';
+				echo $args['after_widget']; //phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+			}
+			return;
+		}
+
+		if ( wp_script_is( 'mailerlite-subscriber-popup', 'enqueued' ) ) {
+			return;
+		}
+
+		wp_register_script( 'mailerlite-universal', 'https://static.mailerlite.com/js/universal.js', array(), '20200521', true );
+		wp_enqueue_script(
+			'mailerlite-subscriber-popup',
+			plugins_url( 'subscriber-popup.js', __FILE__ ),
+			array( 'mailerlite-universal' ),
+			'20200521',
+			true
+		);
+		wp_localize_script(
+			'mailerlite-subscriber-popup',
+			'jetpackMailerliteSettings',
+			array(
+				'account' => esc_attr( $instance['account'] ),
+				'uuid'    => esc_attr( $instance['uuid'] ),
+			)
+		);
+
+	}
+
+	/**
+	 * Updates a particular instance of a widget.
+	 *
+	 * @param array $new_instance New settings for this instance as input by the user via WP_Widget::form().
+	 * @param array $old_instance   Old settings for this instance.
+	 *
+	 * @return mixed
+	 */
+	public function update( $new_instance, $old_instance ) { // @phpcs:ignore Generic.CodeAnalysis.UnusedFunctionParameter.FoundInExtendedClassAfterLastUsed
+		$instance['account'] = wp_kses( stripslashes( $new_instance['account'] ), array() );
```

Comment:
@artpi  We were testing the FSE 1.9 release today (D44727-code) and @p-jackson noticed that this introduces a php lint error which is blocking commit. 

`$instance is an undefined variable.`

I think the deploy will go ahead as it doesn't cause PHP errors. Just leaving a note here 👍

Reply:
Fixing in https://github.com/Automattic/wp-calypso/pull/43176

Reply:
Thank you!